pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -112,7 +112,6 @@ in stdenv.mkDerivation {
     "-Dhostnamed=true"
     "-Dnetworkd=true"
     "-Dportabled=false"
-    "-Dremote=false"
```

Comment:
This should be set explicitly to true, not implicitly use defaults.

Reply:
Done.